assets/prototype/domain/shared/predicates/tickets/isPending.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { differenceInMinutes } from 'date-fns';
+import { now } from './filters';
+
+/**
+ * @function
+ * @param {Object} ticket model object
+ * @param {boolean} includeArchived if true will not filter out archived entities
```

Comment:
this jsDoc can be removed since the parameter does not exist

Reply:
updated, thx